src/github.com/couchbase/sync_gateway/rest/server_context.go

```diff
@@ -632,3 +650,30 @@ func collectAccessRelatedWarnings(config *DbConfig, context *db.DatabaseContext)
 	return []string{}
 
 }
+
```

Comment:
Does this need to be a global function, as opposed to a method on DatabaseContext?